packages/atomic/cypress/e2e/pager.cypress.ts

```diff
@@ -127,4 +127,45 @@ describe('Pager Test Suites', () => {
       CommonAssertions.assertContainsComponentError(PagerSelectors, true);
     });
   });
+
+  describe('Options for previous next button icons', () => {
+    before(() => {
+      new TestFixture().with(addPager()).init();
+    });
+
+    it('should expose shadow part for next icon', () => {
+      cy.get('atomic-pager').shadow().find('[part="next-button-icon"]').click();
+      PagerAssertions.assertRenderPager(2);
```

Comment:
This should probably be extracted outside of this `it` block, since it creates another `it` block inside the current `it` block.